src/style.css

```diff
@@ -0,0 +1,91 @@
+@import "./blocks/variables.css";
+@import "./blocks/normalize.css";
+@import "./blocks/stars.css";
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
```

Comment:
it's not a common rules, remove it from the html selector and set only where you really need it